site/content/integrate/webhooks/outgoing/_index.md

```diff
@@ -97,6 +97,7 @@ Outgoing webhooks support more than just the `text` field. Here is a full list o
 | `attachments`   | [Message attachments]({{<ref "/integrate/reference/message-attachments">}}) used for richer formatting options.                                                                                                                                                                                                                                                                                                                                                                                                           | If `text` is not set, yes        |
 | `type`          | Sets the post `type`, mainly for use by plugins.<br/>If not blank, must begin with "`custom_`".<br/>Specifying a value for the `attachments` property will cause this field to be ignored, and the `type` value set to `slack_attachment`.                                                                                                                                                                                                                                                                                | No                               |
 | `props`         | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br/>Mainly used by other integrations accessing posts through the REST API.<br/>The following keys are reserved: `from_webhook`, `override_username`, `override_icon_url`, `webhook_display_name`, and `attachments`.                                                                                                                                                                                                              | No                               |
+| `priority`      | Set the priority of the message. See [Message Priority](/integrate/reference/message-priority/)                                                                                                                                                                                                                                                                                                                                                                                                                           | No                               |
```

Comment:
I don't understand why we have the "required" column in outgoing webhooks.

Reply:
@fmartingr - I don't either. @mickmister - Do you know why the required column is needed and what it is trying to communicate?

Reply:
Yeah doesn't make sense to me either